### PR TITLE
session: correct the function name in the comment

### PR DESCRIPTION
**Pull request by qpakzk**
I found some typos in comments and corrected them.

### CODE REVIEW

File: utils/fstack.c [hunk 1]
```diff
@@ -503,7 +503,7 @@ int fstack_setup_filters(struct uftrace_opts *opts, struct uftrace_data *handle)
 
 /**
  * fstack_get - retrieve func_stack entry in a task
- * @task - tracee task
+ * @task - trace task
```
Comment (honggyukim) [inline utils/fstack.c:506]: Ah. `tracee` might be correct and used on purpose. @namhyung Could you please confirm this?

Comment (namhyung) [reply utils/fstack.c:506]: Yeah, it was intended.

File: utils/session.c [hunk 2]
```diff
@@ -695,7 +695,7 @@ struct uftrace_symbol * task_find_sym(struct uftrace_session_link *sessions,
 }
 
 /**
- * task_find_sym - find a symbol that matches to @addr
+ * task_find_sym_addr - find a symbol that matches to @addr
```
Comment (honggyukim) [inline utils/session.c:698]: As @namhyung mentioned, `tracee` is intentional, but we can leave this change. So the title of the commit is better to be changed.

File: utils/fstack.c (hunk 1 above)
Comment (qpakzk) [reply utils/fstack.c]: I understand that `tracee` is the opposite of `tracer`.

File: utils/session.c (hunk 2 above)
Comment (qpakzk) [reply utils/session.c:698]: @honggyukim I changed my commit. I appreciate it.

**Review by namhyung — APPROVED** [review]
LGTM

**Pull request merged**